src/helpers/gqlConverters.js

```diff
@@ -34,11 +34,19 @@ function usecaseResponse2gql(useCase, presence) {
     return name
 }
 
+const EnumConventions = Object.freeze({
```

Comment:
wrong direction... 

You implemented all the "cases". No! No! you just have allow a function to be passed to override the internal convention (lower case). that is all...

Reply:
although I still think that this could be solved with just the custom name, the function was implemented. We are receiving a function